src/styles/main.scss

```
@@ -1,3 +1,77 @@
+@import "../styles/utils/var.scss";
+@import "../styles/mixin/mixin.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  width: 100%;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: column;
+  gap: $cellGap;
+  padding: $padd;
+  max-width: $weekWidth;
+  justify-content: center;
+  margin-left: 25%;
+  margin-top: 10%;
+}
+
+.week {
+  display: flex;
+  flex-direction: row;
+  gap: $cellGap;
+  flex-wrap: wrap;
+
+  &__days {
+    @include square($size);
+    border: 1px solid black;
+    background-color: $calendColor;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    transition: transform 0.5s;
```

you also change the background color:
```suggestion
    transition: transform 0.5s, background-color: 0.5s;
```
also, it'd be nice to move 0.5s into a variable (e.g. `$transition-duration`)